rpcs3/Emu/RSX/GL/GLTextureCache.h

```diff
@@ -396,7 +396,7 @@ namespace gl
 		 */
 		void destroy()
 		{
-			if (!is_locked() && pbo_id == 0 && vram_texture == nullptr && m_fence.is_empty() && managed_texture.get() == nullptr)
+			if (!is_locked() && pbo_id == 0 && vram_texture == nullptr && m_fence.is_empty() && managed_texture == nullptr)
```

Comment:
I think just managed_texture is enough; smartptr has default bool operator for this.

Reply:
you mean !managed_texture in this case.
and what about vram_texture then?

Reply:
Its not necessary for either of them, but I feel its worse for a smart_ptr since that actually does have an explicit bool. vram_texture is a raw pointer.